release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -123,6 +168,14 @@ You can now deploy a cluster with Secure Boot when using installer-provisioned i
 [id="ocp-4-7-networking"]
 === Networking
 
+[id="ocp-4-7-kuryr-selectors"]
```

Comment:
in case this id will be displayed on the docs page I would recommend rephrasing to ocp-4-7-kuryr-service-without-selectors, other than that it looks great. Thanks.

Reply:
TY!

Reply:
Done. https://github.com/openshift/openshift-docs/pull/28504/commits/a72393ff5f366d706d23705e3a24ebd747b5759a